tests/setup.go

```
@@ -0,0 +1,69 @@
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+func Setup(t *testing.T) (testcontainers.Container, string, error) {
+	container, err := SetupPostgresContainer(t)
+	if err != nil {
+		return nil, "", err
+	}
+
+	connString, err := GetConnString(t, container)
+	if err != nil {
+		container.Terminate(context.Background())
```

golangci-lint

Error return value of `container.Terminate` is not checked

(errcheck)